app.py

```
@@ -249,106 +293,342 @@ def extract_job_info(text):
+            
+            data['linkedin_email'] = linkedin_email
+            data['linkedin_id'] = linkedin_id
+            data['linkedin_username'] = linkedin_username
+            data['profile_picture'] = picture
+            data['access_token'] = access_token
+            data['user_linkedin_pages'] = get_user_organizations(access_token)
+            
+        except Exception as e:
+            return jsonify({"error": f"LinkedIn authentication error: {str(e)}"}), 400
+
+        is_superadmin = email in SUPER_ADMIN_EMAILS if email else False
```

Ruff (0.11.9)

Local variable `is_superadmin` is assigned to but never used

Remove assignment to unused variable `is_superadmin`

(F841)